packaging/linux/flatpak/dev.lizardbyte.app.Sunshine.yml

```diff
@@ -32,6 +32,9 @@ modules:
   # Test dependencies
   - "modules/xvfb/xvfb.json"
 
+  # Build dependencies
+  - "modules/nlohmann_json.json"
```

Comment:
Isn't this a runtime dependency (when dynamic linked)?

Reply:
Nope, it's a header-only library.